packages/client/src/types.ts

```diff
@@ -43,6 +43,10 @@ export interface WorkflowExecutionDescription {
   raw: DescribeWorkflowExecutionResponse;
 }
 
+export interface FetchedHistory {
+  history: proto.temporal.api.history.v1.History;
+}
```

Comment:
might we want more fields on this in future? if not, can we just use `proto.temporal.api.history.v1.History`?

Reply:
Yeah, I think this could potentially end up with more stuff which is why I added it